language/stdlib/modules/SlidingNonce.move

```diff
@@ -72,6 +72,10 @@ module SlidingNonce {
         move_to(account, SlidingNonce {  min_nonce: 0, nonce_mask: 0 });
     }
 
+    public fun holds_sliding_nonce(addr: address) {
```

Comment:
The name looks like the function should return a boolean.
May be it can be called something like `assert_holds_sliding_nonce`?